airflow/utils/email.py

```diff
@@ -47,8 +48,26 @@ def send_email(
     conn_id: str | None = None,
     custom_headers: dict[str, Any] | None = None,
     **kwargs,
-):
-    """Send email using backend specified in EMAIL_BACKEND."""
+) -> None:
+    """
+    Send an email using the backend specified in the `EMAIL_BACKEND` configuration option.
+
+    :param to: A list or iterable of email addresses to send the email to.
+    :param subject: The subject of the email.
+    :param html_content: The content of the email in HTML format.
+    :param files: A list of paths to files to attach to the email.
+    :param dryrun: If `True`, the email will not actually be sent. Default: `False`.
```

Comment:
```suggestion
    :param dryrun: If `True`, the email will not actually be sent. Default: `False`.
```

Either using double backticks or star (italic, matching the style used by the Python stdlib documentation)